embeddings/embedding/static/word2vec.py

```diff
@@ -0,0 +1,54 @@
+from dataclasses import dataclass, field
+from typing import Any, Literal, Optional
+
+from embeddings.embedding.static.config import GensimFileConfig, StaticModelHubConfig
+from embeddings.embedding.static.embedding import SingleFileEmbedding, StandardStaticWordEmbedding
+
+
+@dataclass
+class KGR10Word2VecConfig(GensimFileConfig):
```

Comment:
shouldn't this go to the config.py file?

Reply:
i see the logic now of putting the fasttext/word2vec code together and leaving the general config and embeddings in separate files